### PR TITLE
docs: apisix ingress with cert manager

### PR DESCRIPTION
**Pull request by KID-G**
Changes:

Add blog apisix ingress with cert manager

### CODE REVIEW

**Comment by KID-G** [comment]
Please review this blog @guoqqqi @yzeng25

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+- cert manager
+description: 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
+tags: [Ecosystem]
+---
+
+> 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
+
+<!--truncate-->
+
+> 作者[张晋涛](https://github.com/tao12345666333)，API7.ai 云原生工程师，Apache APISIX Committer。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]: He's a PMC.
```suggestion
> 作者[张晋涛](https://github.com/tao12345666333)，API7.ai 云原生工程师，Apache APISIX PMC。
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+## 步骤一：环境准备
+
+如果你希望按照本文的指导进行实际的操作，请确保以下环境和工具已准备就绪：
+
+1. 准备一个可用的 Kubernetes 集群，开发环境中，你可以使用 [Kind](https://kind.sigs.k8s.io/) 和 [Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/)
+2. 安装 [kubectl](https://kubernetes.io/docs/tutorials/hello-minikube/)
+3. 安装 [Helm v3](https://helm.sh/)
+
+>请注意，下文所有的操作都将在 ingress-apisix 命名空间中执行，因此需要先创建该命名空间：`kubectl create namespace ingress-apisix`
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]:
```suggestion
> 请注意，下文所有的操作都将在 ingress-apisix 命名空间中执行，因此需要先创建该命名空间：`kubectl create namespace ingress-apisix`
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+replicaset.apps/cert-manager-webhook-bc964d98b         1         1         1       45m
+
+NAME                           READY   AGE
+statefulset.apps/apisix-etcd   2/3     57m
+
+NAME                                     COMPLETIONS   DURATION   AGE
+job.batch/cert-manager-startupapicheck   1/1           6m24s      45m
+```
+
+>[Kubernetes Controller Manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) 的机制决定了 Pod 名称会有所不同。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]:
```suggestion
> [Kubernetes Controller Manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) 的机制决定了 Pod 名称会有所不同。
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+  selfSigned: {}
+```
+
+并创建自签名证书颁发者。
+
+```bash
+kubectl apply -f issuer.yaml
+```
+
+>请注意，自签名颁发对象不推荐使用在生产环境中！更多证书颁发对象的配置请参考[这里](https://cert-manager.io/docs/configuration/)。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]:
```suggestion
> 请注意，自签名颁发对象不推荐使用在生产环境中！更多证书颁发对象的配置请参考[这里](https://cert-manager.io/docs/configuration/)。
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+然后为域名 `httpbin.org` 创建一张证书。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md:141]:
```suggestion

然后为域名 `httpbin.org` 创建一张证书。
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+- 云原生
+- APISIX
+- 证书管理
+- Vault
+- cert manager
+description: 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
+tags: [Ecosystem]
+---
+
+> 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]:
```suggestion
> Cert Manager 解决了 Kubernetes 中证书的默认存储方式的痛点，并逐步成为了 Kubernetes 生态中证书签发和管理领域中的事实标准。
```

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md (hunk 5 above)
Comment (yzeng25) [reply blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md:141]: See, if you don't add a new line, it understands the two sentences as whole, which is not the case.

image

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md (hunk 6 above)
Comment (yzeng25) [reply blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]: The quote shows up properly, but the markdown syntax says always add a space between `>`and quoted texts, see 
https://www.markdownguide.org/basic-syntax/#nested-blockquotes.

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md (hunk 1 above)
Comment (KID-G) [reply blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]: Sorry to downgrade him

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md (hunk 5 above)
Comment (KID-G) [reply blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md:141]: yes , it is,  thanks

File: blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md (hunk 6 above)
Comment (KID-G) [reply blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md]: Thank you very much, I will pay attention next time.

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
Can we get a couple more approvals and reviews asap on this one, too @juzhiyuan @guoqqqi @SkyeYoung

**Review by SkyeYoung — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**